File: day1.py
# Advent of code 2018
# Antti "Waitee" Auranen

def main():
    number = 0
    while(True):
        try:
            s = int(input('> '))

        except ValueError:
            print("Frequency is " , number)
            return number
        number += s

main()

File: day1.hs
-- Advent of code 2018 day 1 challenge in haskell
-- by Antti Auranen
import Text.Printf
import Text.Read
import Control.Monad
import Data.Maybe


main = do
	line <- getLine
	let i :: Int
		i = read line
	eval <- replicateM i readLn

	
	
	
eval :: [String] -> Maybe Int
eval x = do
 	printf (fmap sum $ sequence $ x)
